src/tcp.cpp

```diff
@@ -318,46 +318,7 @@ int zmq::tcp_read (fd_t s_, void *data_, size_t size_)
 
 void zmq::tcp_assert_tuning_error (zmq::fd_t s_, int rc_)
```

Comment:
why keeping this function, if the usage changes to the new one from ip.cpp?

Reply:
It's public, so I was not sure, if this function might have been used somewhere outside libzqm. Removed it completely.

Reply:
Only the API in zmq.h is exposed to outside of libzmq